iroh/src/sync_engine.rs

```diff
@@ -243,11 +243,12 @@ impl SyncEngine {
     }
 }
 
-pub(crate) fn entry_to_content_status(entry: EntryStatus) -> ContentStatus {
+pub(crate) fn entry_to_content_status(entry: io::Result<EntryStatus>) -> ContentStatus {
     match entry {
-        EntryStatus::Complete => ContentStatus::Complete,
-        EntryStatus::Partial => ContentStatus::Incomplete,
-        EntryStatus::NotFound => ContentStatus::Missing,
+        Ok(EntryStatus::Complete) => ContentStatus::Complete,
+        Ok(EntryStatus::Partial) => ContentStatus::Incomplete,
+        Ok(EntryStatus::NotFound) => ContentStatus::Missing,
+        Err(_) => ContentStatus::Missing,
```

Comment:
Not sure if it is OK to just return missing on error, but anything else would require major changes in the sync engine.

Reply:
maybe add a debug log at least?